plutus-tx-plugin/src/Language/PlutusTx/Plugin.hs

```diff
@@ -326,3 +326,61 @@ runCompiler opts expr = do
 
     let uplcP = UPLC.eraseProgram plcP
     pure (pirP, uplcP)
+
+-- | Get the 'GHC.Name' corresponding to the given 'TH.Name', or throw an exception if we can't get it.
```

Comment:
```suggestion
-- | Get the 'GHC.Name' corresponding to the given 'TH.Name', or throw an error if we can't get it.
```

Reply:
Once I read about the difference of exceptions VS errors, but in my head they always feel the same :stuck_out_tongue_closed_eyes:

Reply:
It's loose, but in Haskell I think people do generally only use the word "exception" for the real runtime exceptions. "Error" is also vague, but at least it's not "exception" :laughing: